src/main/java/org/folio/auth/authtokenmodule/tokens/TokenType.java

```diff
@@ -0,0 +1,9 @@
+package org.folio.auth.authtokenmodule.tokens;
+
+public final class TokenType {
+  public static final String REFRESH = "refresh";
+  public static final String ACCESS = "access";
+  public static final String API = "api";
+  public static final String MODULE = "module";
+  public static final String DUMMY = "dummy";
+}
```

Comment:
Could also have been const public static member of each Token class implementation to avoid keeping things "in-sync"